### PR TITLE
Updated to Swift 4.2

### PR DESCRIPTION
**Pull request by mohpor**
Signed-off-by: Mohammad Porooshani <porooshani@gmail.com>

### CODE REVIEW

File: Lady/ViewController.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ class ViewController: UIViewController {
         let propose: Propose = {
             proposeToAccess(photos, agreed: {
                 print("I can access Photos. :]\n")
-                if UIImagePickerController.isSourceTypeAvailable(UIImagePickerControllerSourceType.savedPhotosAlbum) {
+                if UIImagePickerController.isSourceTypeAvailable(UIImagePickerController.SourceType.savedPhotosAlbum) {
```
Comment (nixzhu) [inline Lady/ViewController.swift]: Use omitted source type

File: Lady/ViewController.swift [hunk 2]
```diff
@@ -33,7 +33,7 @@ class ViewController: UIViewController {
         let propose: Propose = {
             proposeToAccess(camera, agreed: {
                 print("I can access Camera. :]\n")
-                if UIImagePickerController.isSourceTypeAvailable(UIImagePickerControllerSourceType.camera){
+                if UIImagePickerController.isSourceTypeAvailable(UIImagePickerController.SourceType.camera){
```
Comment (nixzhu) [inline Lady/ViewController.swift]: Use omitted source type

**Comment by mohpor** [comment]
Done

**Comment by nixzhu** [comment]
@mohpor Thanks!

**Pull request merged**